<?xml version="1.0" encoding="UTF-8"?>
<robot generator="Robot 6.1.1 (Python 3.11.6 on win32)" generated="20231205 15:16:15.965" rpa="false" schemaversion="4">
<suite id="s1" name="loginSuites" source="D:\Lisallah\Pelatihan\Sanbercode\Quality Assurance\MobileAutomation_Kel14\suites\loginSuites.robot">
<test id="s1-t1" name="Login Valid Credential" line="14">
<kw name="Open Flight Application" library="base">
<kw name="Open Application" library="AppiumLibrary">
<arg>remote_url=${REMOTE_URL}</arg>
<arg>platformName=${PLATFORM_NAME}</arg>
<arg>platformVersion=${PLATFORM_VERSION}</arg>
<arg>deviceName=${DEVICE_NAME}</arg>
<arg>appPackage=${APP_PACKAGE}</arg>
<arg>appActivity=${APP_ACTIVITY}</arg>
<doc>Opens a new application to given Appium server.
Capabilities of appium server, Android and iOS,
Please check https://github.com/appium/appium/blob/master/docs/en/writing-running-appium/server-args.md
| *Option*            | *Man.* | *Description*     |
| remote_url          | Yes    | Appium server url |
| alias               | no     | alias             |
| strict_ssl          | No     | allows you to send commands to an invalid certificate host like a self-signed one. |</doc>
<kw name="Capture Page Screenshot" library="AppiumLibrary">
<doc>Takes a screenshot of the current page and embeds it into the log.</doc>
<msg timestamp="20231205 15:16:44.589" level="FAIL">No application is open</msg>
<status status="FAIL" starttime="20231205 15:16:44.460" endtime="20231205 15:16:45.314"/>
</kw>
<msg timestamp="20231205 15:16:45.455" level="WARN">Keyword 'Capture Page Screenshot' could not be run on failure: No application is open</msg>
<msg timestamp="20231205 15:16:45.554" level="FAIL">WebDriverException: Message: An unknown server-side error occurred while processing the command. Original error: Could not proxy command to the remote server. Original error: socket hang up
Stacktrace:
UnknownError: An unknown server-side error occurred while processing the command. Original error: Could not proxy command to the remote server. Original error: socket hang up
    at UIA2Proxy.command (C:\Program Files\Appium Server GUI\resources\app\node_modules\appium\node_modules\appium-base-driver\lib\jsonwp-proxy\proxy.js:274:13)
    at processTicksAndRejections (internal/process/task_queues.js:85:5)</msg>
<status status="FAIL" starttime="20231205 15:16:17.506" endtime="20231205 15:16:45.575"/>
</kw>
<status status="FAIL" starttime="20231205 15:16:17.505" endtime="20231205 15:16:45.591"/>
</kw>
<kw name="Click Sign In Button On Home Page" library="homePage">
<status status="NOT RUN" starttime="20231205 15:16:45.594" endtime="20231205 15:16:45.594"/>
</kw>
<kw name="Input Username On Login Page" library="loginPage">
<arg>${VALID_USERNAME}</arg>
<status status="NOT RUN" starttime="20231205 15:16:45.595" endtime="20231205 15:16:45.595"/>
</kw>
<kw name="Input Password On Login Page" library="loginPage">
<arg>${VALID_PASSWORD}</arg>
<status status="NOT RUN" starttime="20231205 15:16:45.596" endtime="20231205 15:16:45.596"/>
</kw>
<kw name="Click Sign In Button On Login Page" library="loginPage">
<status status="NOT RUN" starttime="20231205 15:16:45.598" endtime="20231205 15:16:45.598"/>
</kw>
<kw name="Close Flight Application" library="base">
<status status="NOT RUN" starttime="20231205 15:16:45.599" endtime="20231205 15:16:45.600"/>
</kw>
<status status="FAIL" starttime="20231205 15:16:17.505" endtime="20231205 15:16:45.607">WebDriverException: Message: An unknown server-side error occurred while processing the command. Original error: Could not proxy command to the remote server. Original error: socket hang up
Stacktrace:
UnknownError: An unknown server-side error occurred while processing the command. Original error: Could not proxy command to the remote server. Original error: socket hang up
    at UIA2Proxy.command (C:\Program Files\Appium Server GUI\resources\app\node_modules\appium\node_modules\appium-base-driver\lib\jsonwp-proxy\proxy.js:274:13)
    at processTicksAndRejections (internal/process/task_queues.js:85:5)</status>
</test>
<status status="FAIL" starttime="20231205 15:16:15.991" endtime="20231205 15:16:45.635"/>
</suite>
<statistics>
<total>
<stat pass="0" fail="1" skip="0">All Tests</stat>
</total>
<tag>
</tag>
<suite>
<stat pass="0" fail="1" skip="0" id="s1" name="loginSuites">loginSuites</stat>
</suite>
</statistics>
<errors>
<msg timestamp="20231205 15:16:17.415" level="ERROR">Error in file 'D:\Lisallah\Pelatihan\Sanbercode\Quality Assurance\MobileAutomation_Kel14\pageObjects\homePageObject\homePage.robot' on line 3: Processing variable file 'D:\Lisallah\Pelatihan\Sanbercode\Quality Assurance\MobileAutomation_Kel14\pageObjects\homePageObject\home-locators.yaml' failed: Using YAML variable files requires PyYAML module to be installed. Typically you can install it by running `pip install pyyaml`.</msg>
<msg timestamp="20231205 15:16:17.500" level="ERROR">Error in file 'D:\Lisallah\Pelatihan\Sanbercode\Quality Assurance\MobileAutomation_Kel14\pageObjects\loginPageObject\loginPage.robot' on line 3: Processing variable file 'D:\Lisallah\Pelatihan\Sanbercode\Quality Assurance\MobileAutomation_Kel14\pageObjects\loginPageObject\login-locators.yaml' failed: Using YAML variable files requires PyYAML module to be installed. Typically you can install it by running `pip install pyyaml`.</msg>
<msg timestamp="20231205 15:16:45.455" level="WARN">Keyword 'Capture Page Screenshot' could not be run on failure: No application is open</msg>
</errors>
</robot>
